### PR TITLE
Add explicit or to serializer

### PR DESCRIPTION
**Pull request by fbentz**

### CODE REVIEW

File: src/Elastic/Serializer.elm [hunk 1]
```diff
@@ -1,50 +1,58 @@
...
+module Elastic.Serializer exposing (Config(..), run)
 
 import Elastic.Expression exposing (Expr(..))
 
 
-group : Expr -> String
-group expr =
-    "(" ++ run expr ++ ")"
+type Config
+    = Config Bool
```
Comment (n1k0) [inline src/Elastic/Serializer.elm]: Why not a record? what if we want to add more options later on? I'd suggest we keep the opaque type though.

File: tests/SerializerTest.elm [hunk 2]
```diff
@@ -6,41 +6,46 @@ import Expect exposing (Expectation)
 import Test exposing (..)
 
 
+serializeExpr_ : Expr -> String
+serializeExpr_ expr =
+    serializeExpr { explicitOr = False } expr
```
Comment (n1k0) [inline tests/SerializerTest.elm]: I think we should run two different test suites, one with the option enabled and the other without.

**Comment by fbentz** [comment]
Done for spaces.

File: src/Elastic.elm [hunk 3]
```diff
@@ -37,7 +37,15 @@ parseQuery rawQuery =
 
 
 {-| serialize an AST into simple query string for elastic search
+
+Nit: explicitOr bool parameter allow to serialize explicitly Or Expr with parenthis.
```
Comment (n1k0) [inline src/Elastic.elm]: Why Nit?

**Pull request merged**